packages/chain-adapters/src/utxo/utxoSelect/index.ts

```diff
@@ -44,5 +44,14 @@ export const utxoSelect = (input: UTXOSelectInput) => {
     return coinSelect<unchained.bitcoin.Utxo>(utxos, outputs, Number(input.satoshiPerByte))
   })()
 
+  if (input.from && result?.outputs?.[1]?.value) {
```

Comment:
```suggestion
  /**
   * note - the 0-indexed output is the `to` address, and if `input.from` is included as an argument, the
   * 1-indexed output will be the change address, which we overwrite below
   */
  if (input.from && result?.outputs?.[1]?.value) {
```

Reply:
comments explaining implicit logic - this bit i had to scrutinize a little to understand what's going on. looks good tho!